drop-in/src/main/java/com/adyen/checkout/dropin/internal/ui/PaymentMethodListDialogFragment.kt

```diff
@@ -214,30 +215,15 @@ internal class PaymentMethodListDialogFragment :
                 paymentMethodsListViewModel.onClickConfirmationButton()
             }
 
-        when (storedPaymentMethodModel) {
-            is StoredCardModel -> {
-                dialog.setMessage(
-                    requireActivity().getString(
-                        R.string.last_four_digits_format,
-                        storedPaymentMethodModel.lastFour,
-                    ),
-                )
-            }
-
-            is GenericStoredModel -> {
-                // do nothing
-            }
-
-            is StoredACHDirectDebitModel -> {
-                dialog.setMessage(
-                    requireActivity().getString(
-                        R.string.last_four_digits_format,
-                        storedPaymentMethodModel.lastFour,
-                    ),
-                )
-            }
+        val message = when (storedPaymentMethodModel) {
+            is StoredCardModel ->
+                requireActivity().getString(R.string.last_four_digits_format, storedPaymentMethodModel.lastFour)
```

Comment:
Do we need to use `requireActivity()` here? I think you have access to the current context right?

Reply:
Here also using `popUpMessage` now.